rag/flow/chunker/token_chunker.py

```
@@ -359,31 +362,79 @@ async def _invoke(self, **kwargs):
+                            segments.append((seg, seg_start, seg_end))
+                        pos = seg_end
+                        if i + 1 < len(raw):
+                            pos += len(raw[i + 1])
+                else:
+                    segments = [(combined_text, 0, len(combined_text))]
+
+                for text, seg_start, seg_end in segments:
+                    if not text.strip():
+                        continue
+                    seg_pos = []
+                    for (istart, iend), item_pos in zip(item_ranges, text_buffer_pos):
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)